breez/breez.proto

```diff
@@ -19,7 +19,9 @@ service FundManager {
     rpc AddFund (AddFundRequest) returns (AddFundReply) {}
     rpc AddFundStatus (AddFundStatusRequest) returns (AddFundStatusReply) {}
     rpc RemoveFund (RemoveFundRequest) returns (RemoveFundReply) {}
-    rpc GetPayment (GetPaymentRequest) returns (GetPaymentReply) {}    
+    rpc GetPayment (GetPaymentRequest) returns (GetPaymentReply) {}
+    rpc PaySwapInvoice (PaySwapInvoiceRequest) returns (PaySwapInvoiceReply) {}
+    rpc GetRefunds (GetRefundsRequest) returns (GetRefundsReply) {}
```

Comment:
Also here we need to gain this information in the client LND node instead of contacting the server.